utils/fetch-musicbrainz-data/fetch.go

```diff
@@ -0,0 +1,389 @@
+package main
```

Comment:
typically we would make these kind of scripts available as a "mu script", see https://github.com/mu-semtech/simple-script-store and https://github.com/mu-semtech/mu-cli#writing-your-own-script for more info

Reply:
ah, didn't know about that. will keep it in mind for the next time!